src/api/finnKandidatApi.ts

```diff
@@ -1,19 +1,36 @@
 import Kandidat from '../types/Kandidat';
 import api from './initialize';
+import { ArbeidstidBehov } from '../types/Behov';
+import { Omit } from 'react-router';
 
-const parseSistEndretDato = (kandidat: Kandidat) => {
+type KandidatDataTransfer = Omit<Kandidat, 'arbeidstidBehov'> & {
```

Comment:
arbeidstidBehov har blitt en liste, så har to funksjoner for å konvertere kandidat før innsending og ved uthenting

Reply:
KandidatDTO